src/RuleCommonITILObject.php

```diff
@@ -469,6 +511,20 @@ public function executeActions($output, $params, array $input = [])
                                     "name" => $regex_value,
                                 ]);
 
+                                // Stop at the first valid category found
+                                if ($result) {
+                                    $output['itilcategories_id'] = $category->getID();
+                                    break;
+                                }
+                            }
+                        } elseif ($field == "itilcategories_id_by_completename") {
```

Comment:
It seems that naming convention is to prefix by a `_` when action does not match an existing field of the item.

```suggestion
                        } elseif ($field == "_itilcategories_id_by_completename") {
```

Reply:
you're right, fixed !